code/modules/events/rogue/_rogue.dm

```diff
@@ -34,19 +34,19 @@ GLOBAL_LIST_INIT(badomens, list())
 	var/used
 	switch(eventreason)
 		if(OMEN_ROUNDSTART)
-			used = "Zizo."
+			used = "Levishth."
 		if(OMEN_NOLORD)
-			used = "The Monarch is dead! We need a new ruler."
+			used = "The Duke is dead! We need a new ruler."
```

Comment:
Why

Reply:
Every other mention is Duke. Also doesn't matter since king/duke/monarch/whatever is shelved.